ando/tools/generator/utils.py

```diff
@@ -0,0 +1,84 @@
+from os import path
```

Comment:
I would be in favor of also using pathlib here as I think it makes the consistent handling of pathes across different platforms easier and we are already using it in other submodules of this package.

Reply:
done